deploy/docker/dev.web.Dockerfile

```diff
@@ -8,5 +8,4 @@ RUN apt update; apt install -y cmake
 COPY requirements.txt src/plugins/requirements-*.txt /tmp/
 RUN ls /tmp/requirements*.txt | xargs -i,, pip --no-cache-dir install -r ,,
 
-# ./src is expected to be mounted with a docker volume
-CMD ["uvicorn", "app:app", "--host", "0.0.0.0", "--port", "5000", "--reload"]
+CMD pip install -e /usr/src/app && uvicorn mquery.app:app --host 0.0.0.0 --port 5000 --reload
```

Comment:
This is only a dev dockerfile, we need to install the pip package in editable mode to have hot reload.